samples/publish-confirm/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "publish-confirm",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "parcel src/index.html"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@contentful/forma-36-fcss": "0.0.9",
+    "@contentful/forma-36-react-components": "^2.3.0",
+    "babel-polyfill": "^6.26.0",
+    "contentful-ui-extensions-sdk": "^3.5.0",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
+    "relative-date": "^1.1.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-proposal-class-properties": "^7.3.0",
+    "cssnano": "^4.1.8",
```

Comment:
this is not used. I also think parcel doesn't require `@babel` dependencies you've declared.